client/cmdlfhid.c

```diff
@@ -400,7 +400,7 @@ static int CmdHIDBrute(const char *Cmd) {
                 param_getstr(Cmd, cmdp + 1, format, sizeof(format));
                 format_idx = HIDFindCardFormat(format);
                 if (format_idx == -1) {
-                    PrintAndLogEx(WARNING, "Unknown format: %s", format);
+                    PrintAndLogEx(WARNING, _YELLOW_("Unknown format: %s"), format);
```

Comment:
how about just coloring the param part '%c' , not the text.   Makes it easy to spot the important part.